src/ui/Graphs/RelationshipGraph.tsx

```diff
@@ -71,24 +72,25 @@ export function RelationshipGraph<T extends GraphDataGenericConstraint>({
     edges
   );
 
+  const { nodes: nodesAnimated } = useAnimatedNodes(expandNodes, {
+    animationDuration: 300
+  });
+
   const centerNodeToViewport = useCallback(
     (node: Node<T>) => {
       // when the user interacts with the node, disable auto-aligning of the
       // nodes
       setIsInitialLayoutSetupDone(true);
 
       if (node.data.expandable) {
-        const { x, y, width, height } = getNodesBounds([node]);
-
-        // Calculate the center of the node
-        const nodeCenterX = x + width / 2;
-        const nodeCenterY = y + height / 2;
-
-        // Move the viewport to the center of the node
-        setCenter(nodeCenterX, nodeCenterY, { duration: 800, zoom: getZoom() });
+        fitView({
```

Comment:
We don't want to fit view - Node sizes should only be changed by the user, it should center on the node